code4lib/2019.code4lib.org
# Preconference speaker data & photos

testing plan:

- open up `/workshops/`, check _every workshop_ to ensure that each has at least one speaker listed
- open up `/speakers/`, check that ever speaker has a workshop associated with them (click to expand)
- there should be 32 speakers in total, 29 of whom have images
- images should look OK (not be stretched to look odd—we need vaguely square dimensions for them)
